addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -224,7 +225,11 @@ var ImageWidget = MediaWidget.extend({
     _adaptLoadMore: function () {
         var noMoreImgToLoad = this.IMAGES_DISPLAYED_TOTAL >= this.records.length;
         this.$('.o_load_more').toggleClass('d-none', noMoreImgToLoad);
-        this.$('.o_load_done_msg').toggleClass('d-none', !noMoreImgToLoad);
+        if (this.records.length > 0) {
+            this.$('.o_load_done_msg').toggleClass('d-none', !noMoreImgToLoad);
+        } else {
+            this.$('.o_load_done_msg').toggleClass('d-none', true);
```

Comment:
`this.$('.o_load_done_msg').toggleClass('d-none', !this.records.length || !noMoreImgToLoad)` ?